### ISSUE TITLE
MineralResources sourceReference problem

### ISSUE THREAD

**Issue by CLinGBA** [INSPIRE-MIF/helpdesk#58]
Dear all,
 
There seems to be an issue with EarthResource and the sourceReference element in MineralResources, defined in the UML as: sourceReference: DocumentCitation[1..*]
 
In the XSD, the reference to the DocumentCitation dataType gets lost (only mentioned in the appinfo tag), while the xlinks provided via the gml:AssociationAttributeGroup aren't of help as DocumentCitation is a dataType, thus cannot be referenced.
 
This is an issue, as we thus cannot provide this mandatory information. XSD Snippet showing the issue below

`<element name="sourceReference" nillable="true" maxOccurs="unbounded">
    <annotation>
        <documentation>-- Definition --
The source or reference for the Earth Resource.</documentation>
        <appinfo>
            <targetElement xmlns="http://www.opengis.net/gml/3.2">base2:DocumentCitation</targetElement>
        </appinfo>
    </annotation>
    <complexType>
        <complexContent>
            <extension base="gml:AbstractMemberType">
                <sequence/>
                <attributeGroup ref="gml:AssociationAttributeGroup"/>
            </extension>
        </complexContent>
    </complexType>
</element>`
 
Many thanks,
 
Christian

**Comment by JohannaOtt** [comment]
Hi Christian,

There was a discussion on this in the INSPIRE Forum back in the days https://wayback.archive-it.org/12090/20210123101226/https://inspire.ec.europa.eu/forum/discussion/view/269595/mineraloccurencesourcereference-does-not-allow-inline-creation-of-documentcitation
Kathi wanted to forward the issue to the JRC.


All the best

Johanna

**Comment by sMorrone** [comment]
Dear @CLinGBA 
bugs in current version of INSPIRE schemas (as well as change proposals) should now be reported via the [issue tracker](https://github.com/INSPIRE-MIF/application-schemas/issues) of the [application-schemas github repository]( https://github.com/INSPIRE-MIF/application-schemas) using the **New issue** template.
Once this is done, the change proposal (in this case the bug-fixing proposal) will follow the specific governance and release process for INSPIRE schemas described in [this document](https://github.com/INSPIRE-MIF/application-schemas/blob/main/governance-release-process/process.md) and, if endorsed, will be carried out and lead to a new schema version.
I am now moving this issue to abovementioned application-schemas repository.
In order to help the governance and release process move forward, please fill out the change proposal template to the best of your knowledge.  
Thank you
